encrypt/encrypt.go

```diff
@@ -75,14 +82,41 @@ func Encrypt(args []string) error {
 		return err
 	}
 
-	// no key provided, check for one in the session file
-	if len(publicKeyFileList) == 0 {
+	var pubKey string
 
-		sesKey, err := helpers.GetPublicKey()
+	if publicKeyFileList != nil && *target != "" {
+		return errors.New("only one of -key or -target can be used")
+	}
+
+	if publicKeyFileList == nil && *target != "" {
```

Comment:
```suggestion
	if *target != "" {
```

Since only one of them can be set at this point we only need to check the one we are interested in.

Reply:
No, I think we still need that, otherwise we will get in this block in the case of having set the config as an argument.

Reply:
Don't speculate.
If you are unsure, add a test case in the relevant test file.

Reply:
My phrasing was misleading.
The clause is correct, the tests actually caught it.

Reply:
There are no relevant test cases in the code right now.
And since `publicKeyFileList` is supposed to be `nil` here we don't need to test for it, if it hadn't been nil it would have been caught on line 82.